package-lock.json

```diff
@@ -4806,19 +4806,26 @@
       }
     },
     "gl-line3d": {
-      "version": "1.1.6",
-      "resolved": "https://registry.npmjs.org/gl-line3d/-/gl-line3d-1.1.6.tgz",
-      "integrity": "sha512-22DcHvezFTJ0BK1lYyV9FRV4Z2moey0RAiFynGEIrvbUq3EBd7e+Sftv1/A6kxNUqdp5SIWmMdGznoAPD9P8FQ==",
+      "version": "1.1.8",
+      "resolved": "https://registry.npmjs.org/gl-line3d/-/gl-line3d-1.1.8.tgz",
+      "integrity": "sha512-rTN7JR/F9soPOa35/Tn1bEsly/lEyDRXPftMVlk3iL1vnZUAlt2DtszOzxvBMhCmAk2RdfWqZ5DkciRikXVpng==",
       "requires": {
-        "binary-search-bounds": "^1.0.0",
+        "binary-search-bounds": "^2.0.4",
         "gl-buffer": "^2.0.8",
         "gl-shader": "^4.2.1",
         "gl-texture2d": "^2.0.2",
         "gl-vao": "^1.1.3",
-        "glsl-out-of-range": "^1.0.3",
+        "glsl-out-of-range": "^1.0.4",
         "glsl-read-float": "^1.0.0",
-        "glslify": "^6.1.0",
+        "glslify": "^7.0.0",
         "ndarray": "^1.0.16"
+      },
+      "dependencies": {
+        "binary-search-bounds": {
```

Comment:
Could you check `npm ls binary-search-bounds`? I think I tried using `binary-search-bounds@2.x` in gl3d, but I don't think that worked out. We should double-check.

Reply:
@archmoj this here is now the only potentially-blocking item.

What are the differences between `binary-search-bounds` v1.x and v2.x?

Reply:
I think we may leave that for another PR. It seems that I don't have permission to patch and publish those modules...

Reply:
https://github.com/mikolalysenko/binary-search-bounds/commit/d047eeeff4cb388d423b017b8f3100dda5adefa6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2
This may be relevant to the difference between version 1 and 2.
`ndarray`s are not supported in Version 2.